x-pack/docs/en/rest-api/security/saml-prepare-authentication-api.asciidoc

```diff
@@ -77,17 +77,19 @@ POST /_security/saml/prepare
   "realm" : "saml1"
 }
 --------------------------------------------------
+// TEST[skip:Additional configuration needed in yml files]
```

Comment:
Lets not skip tests without a valid reason. All we need to do here is to change 

```
  setting 'xpack.security.authc.realms.saml.saml1.sp.acs', 'https://kibana.org/api/security/v1/saml'
```

to 

```
  setting 'xpack.security.authc.realms.saml.saml1.sp.acs', 'https://kibana.org/api/security/saml/callback'
```

in x-pack/docs/build.gradle

Reply:
@jkakavas, thanks for the pointer! I wasn't sure how to fix the tests that were breaking the build. I'll create a new PR for updating `build.gradle`.